packages/amplify-category-api/commands/api/add-datasource.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
          */
         const currEnv = amplify.getEnvInfo().envName;
         const teamProviderInfoFilePath = amplify.pathManager.getProviderInfoFilePath();
-        const teamProviderInfo = JSON.parse(fs.readFileSync(teamProviderInfoFilePath));
+        const teamProviderInfo = context.amplify.readJsonFile(teamProviderInfoFilePath);
```

Comment:
For consistency, use `amplify` directly without the `context.`

Reply:
Lets do that later. If there are any problem in the `ServiceMetaData` the CLI will fail to start as they are at the top level

Reply:
I see your point, but don't you think it's better that this is consistent across the whole project? currently, some are on top of the file direct using JSON.parse(fs.readFileSync(.)) directly, such as `amplify-category-analytics/commands/analytics/add.js'; and some are inside the run method using context.amplify.readJsonFile(.), such as `amplify-category-auth/commands/auth/enable.js`

Reply:
Do you think that change belong in this PR? Each plugin has its own style and we could put effort to make them consistent. But should we bundle it here?